keras/applications/applications_test.py

```diff
@@ -137,6 +137,17 @@ def test_application_variable_input_channels(self, app, last_dim):
       self.assertShapeEqual(output_shape, (None, None, None, last_dim))
     backend.clear_session()
 
+  @parameterized.parameters(MODEL_LIST)
+  def test_application_models_compute_output_shape(self, app, last_dim):
+    input_shape = (None, None, None, 4)
+    model = app(weights=None, include_top=False)
+    output_shape = model.compute_output_shape(input_shape)
+    if 'MobileNetV3' in app.__name__:
```

Comment:
This special case should no longer exist since we fixed the `include_top=False` issue for this model (cc @haifeng-jin )

Reply:
It is a compute_output_shape test for all models in applications,
I'm not sure if I need to test all models , or just test the model mentioned in this [issue](https://github.com/keras-team/keras/issues/15282).
Moreover, maybe this test should no longer exist since we already have a test for `TFOpLambda `